frontend/components/YourWalletPage/YourAgent.tsx

```diff
@@ -232,3 +233,10 @@ export const YourAgentWallet = () => {
     </Card>
   );
 };
+
+export const YourAgentWallet = () => (
+  <>
+    <YourAgentWalletBreakdown />
+    <WithdrawFunds />
```

Comment:
should this be visible after withdrawal? i.e where there are no funds?

Reply:
Yes we could leave it as is. It will be just with 0 funds and the user will have nothing to withdraw.